src/library/scala/collection/SortedMap.scala

```diff
@@ -174,7 +174,7 @@ trait SortedMapOps[K, +V, +CC[X, Y] <: Map[X, Y] with SortedMapOps[X, Y, CC, _],
   override def concat[V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = sortedMapFactory.from(new View.Concat(toIterable, xs))
 
   /** Alias for `concat` */
-  @`inline` override final def ++ [V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = concat(xs)
+  override def ++ [V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = concat(xs)
```

Comment:
We should make `WithDefault[K, V]` extend `MapOps[K, V, WithDefault, WithDefault[K, V]]`, so that this alias can stay final.

Reply:
The problem with that approach though is then you need to make a `MapFactory[WithDefault]` which I don't think is possible because then you need to be able to construct an empty WithDefault from nothing.

Reply:
It was actually my initial approach :-).

Reply:
Ah, yes, I forgot about that. Indeed, transformation operations can not preserve the default value.

Reply:
You're saying it's worth giving it up just for the sake of inlining?